ansible/mnesia_snapshot_restore.yml

```diff
@@ -36,9 +36,9 @@
       tags: [health-check]
 
     - name: Restore Mnesia database
-      command: "{{ aeternity_bin }} eval '{atomic, _} = mnesia:restore(\"/tmp/backup\", [{default_op, recreate_tables}]).'"
+      command: "{{ aeternity_bin }} eval 'ok = mnesia:install_fallback(\"/tmp/backup\").'"
```

Comment:
I'm a bit confused. I see the default callback is [`mnesia_backup` in the docs](http://erlang.org/doc/man/mnesia.html#install_fallback-2)